<launch>
  <arg name="disble_torque" default="True" />
  
  <group ns="bender">
    <node name="r_arm_dynamixel_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
      <rosparam file="$(find bender_arm)/config/r_arm_port.yaml" command="load"/>
  	</node>
  	
  </group>

</launch>